content/en/docs/getting-started/workload.md

```diff
@@ -30,16 +30,8 @@ spec:
   runtimeClassName: kata-qemu-coco-dev
 ```
 
-Setting the `runtimeClassName` is usually the only change needed to the pod yaml, but some platforms
-support additional annotations for configuring the enclave. See the [guides](../guides) for
-more details.
-
-With Confidential Containers, the workload container images are never downloaded on the host.
-For verifying that the container image doesn’t exist on the host, you should log into the k8s node and ensure the following command returns an empty result:
-```
-root@cluster01-master-0:/home/ubuntu# crictl -r unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
-```
-You will run this command again after the container has started.
+For the most basic workloads, setting the `runtimeClassName` and `runtime-handler` annotation is usually
```

Comment:
I've seen this annotation being used but I cannot think of a reason why it's needed. Is there any? I've never used it. Should it be s/and/or/?

Reply:
I think you can get away without the annotation, but things will work a little bit better with it. For instance, I think that containerd will pull the full image on the host without it but if you use the annotation it will only pull the manifest. I don't understand this part of the project very well tho tbh.